README.md

```diff
@@ -223,6 +224,16 @@ endfunction
 autocmd! User FzfStatusLine call <SID>fzf_statusline()
 ```
 
+Installation Tips
+-----------------
+
+### dein.vim
+
+```
+call dein#add('junegunn/fzf', { 'build': './install', 'merged': 0 })
```

Comment:
README has this line "Also note that fzf currently does not compile on Windows."